tests/DCPS/XTypes/XTypes.mpc

```diff
@@ -0,0 +1,31 @@
+project(Subscriber): dcps_test, dcps_rtps_udp, msvc_bigobj, dcps_transports_for_test, opendds_optional_security {
```

Comment:
In general, keeping each project in its own directory is the recommended "best practice."  It is not a strict requirement.

However in this case there are now two projects that can generate the same files (from Common.idl).  To do this correctly with parallel builds, add an `after` to one of the projects so that they can't run concurrently.

The project names must be changed since they need to be unique across the entire workspace.